docs/man/manpage_out/helics_app-broker.1

```diff
@@ -0,0 +1,268 @@
+'\" t
```

Comment:
why is this one in troff format?  Is that just a trial?

Reply:
The manpage_out subfolder has a copy of the generated troff files so that there's something that can be installed using `make install`  or for packaging without a dependency on installing a2x.